platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/OrganizationMemberService.java

```diff
@@ -0,0 +1,64 @@
+package uno.cod.platform.server.core.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uno.cod.platform.server.core.domain.Organization;
+import uno.cod.platform.server.core.domain.OrganizationMember;
+import uno.cod.platform.server.core.domain.OrganizationMemberKey;
+import uno.cod.platform.server.core.domain.User;
+import uno.cod.platform.server.core.dto.organizationMember.OrganizationMemberCreateDto;
+import uno.cod.platform.server.core.repository.OrganizationMemberRepository;
+import uno.cod.platform.server.core.repository.OrganizationRepository;
+import uno.cod.platform.server.core.repository.UserRepository;
+
+import javax.transaction.Transactional;
+
+@Service
+@Transactional
+public class OrganizationMemberService {
+    private final OrganizationMemberRepository repository;
+    private final UserRepository userRepository;
+    private final OrganizationRepository organizationRepository;
+
+    @Autowired
+    public OrganizationMemberService(OrganizationMemberRepository repository, UserRepository userRepository, OrganizationRepository organizationRepository) {
+        this.repository = repository;
+        this.userRepository = userRepository;
+        this.organizationRepository = organizationRepository;
+    }
+
+    public void save(OrganizationMemberCreateDto dto) {
```

Comment:
if the user is already member, should we throw an error? currently it would work this way and just continues without an error, since we're using a Set and it does not modify anything (except if we toggle the admin flag) - the question is do we want an error here?